config/routes.rb

```diff
@@ -1,10 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
 
   get 'pages/home'
-  get '/sign_up', to: 'pages#sign_up'
```

Comment:
Since we removed this route you will want to update the path on the link to on the pairs/index page. You can replace the path with '#'. I shouldn't have added any functionality to the buttons at this point.